ch5-catdir/controller.cpp

```diff
@@ -42,11 +61,11 @@ int Controller::run()
 
 std::ostream& Controller::error()
 {
-    m_instance->m_exitStatus = EXIT_FAILURE;
-    return m_instance->m_err << m_instance->m_progName << ": ";
+    m_instance->m_exitStatus = ExitStatus::FAILURE;
+    return m_instance->m_err << m_instance->m_args.m_progName << ": ";
 }
 
-void Controller::process(const std::string_view& path) try
+void Controller::process(const std::string_view& path, bool isArgument) try
```

Comment:
`isArgument` sounds a bit strange to me. Everything is an argument when passed to a function... :)
Maybe `isCmdLineArgument`? I'm not sure myself